components/compositing/compositor.rs

```diff
@@ -1342,8 +1342,12 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
         {
             return false;
         }
-
-        self.update_after_zoom_or_hidpi_change();
+        for top_level_browsing_context_id in self.webviews.painting_order.clone() {
+            self.move_resize_webview(
```

Comment:
The size of webview should be determined by embedder instead of compositor. A dialog webview might want to stick to fixed size event when the window is resized. Could you send `MoveResizeWebView` in simpleservo?

Reply:
@wusyong Could you re-review this PR? It seems that merging the PR is blocked on resolving your change request.